rosette/h/Heap.h

```diff
@@ -251,11 +240,7 @@ extern void* palloc6(unsigned, void*, void*, void*, void*, void*, void*);
 
 static const int alignmentmask = 3;
 
-inline int align(int size) { return ((size + alignmentmask) & ~alignmentmask); }
-
-inline bool Heap::is_new(Ob* p) {
-    return ((void*)p >= newSpaceBase && (void*)p < newSpaceLimit);
-}
+int align(int size);
```

Comment:
This doesn't fit with the theme of the rest. It seems like it should be OK to leave the align function inline.

Reply:
Part of the purpose of this was to remove all use of inline.

Reply:
Well, you've left all the member functions as inline (by virtue of being in the header). You've just removed the inline keyword.

These aren't methods, and so you're actually changing something. If they are actually supposed to be inline functions, the correct solution would be to leave the definition in the header, and remove all the other definitions.

I don't really care if they're inline, though, they just didn't seem to fit what you were doing in the rest of the PR.